core/templates/dev/head/components/SearchBarDirective.js

```diff
@@ -21,11 +21,11 @@ oppia.directive('searchBar', [function() {
     restrict: 'E',
     templateUrl: 'components/searchBar',
     controller: [
-      '$scope', '$rootScope', '$timeout', '$window', 'searchService',
-      'oppiaDebouncer', 'ExplorationCreationButtonService', 'urlService',
-      'CATEGORY_LIST',
+      '$location', '$scope', '$rootScope', '$timeout', '$window',
```

Comment:
Bit of a hidden convention, but we always seem to put $scope first in these things Maybe put $location after $window instead?